src/Components/Form/index.jsx

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import { useState } from 'react';
 import './Form.scss';
 
 function Form (props) {
+    const [url, setUrl] = useState('');
+    const [method, setMethod] = useState('GET');
+    const [json, setJson] = useState('');
```

Comment:
You've got the state to track it and the handle submit to pass it along, but you are missing the text area to input your json.